assignment2/q2.py

```diff
@@ -0,0 +1,29 @@
+# Design an algorithm and write code to find the lowest common ancestor of two nodes in a binary tree.
+#Avoid storing additional nodes in a data structure.
+class Node:
+ 
+
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+ 
+
+
+    def lca(root, n1, n2):
+     
```

Comment:
I really like your minimalistic way of coding! But this solution seems to only work for binary search trees?